rpcsx-os/orbis-kernel-config/orbis-config.hpp

```diff
@@ -103,6 +103,8 @@ inline uint64_t readRegister(void *context, RegisterId id) {
     return c->gregs[REG_RSP];
   case RegisterId::rflags:
     return c->gregs[REG_EFL];
+  default:
+    return 0;
```

Comment:
It is an exceptional case. We cannot continue work if not all variants of enum handled in switch
std::abort() with optional message to stderr before looks better here

Reply:
@DHrpcs3 thanks, done i've added the following:
```
  default:
    std::fprintf(stderr, "***ERROR*** Unhandled RegisterId %d\n", static_cast<int>(id));
    std::abort();
```
i've tried to follow some already defined error messages